src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -6,6 +6,8 @@
 from azure.cli.core.util import CLIError
 from azure.cli.core.commands.parameters import get_resources_in_subscription
 
+from azure.mgmt.containerregistry.v2017_03_01.models import SkuTier
```

Comment:
Some API versions may be missing in Azure Stack or different cloud. Will the command fail in those environments?